code/game/turfs/simulated/walls.dm

```diff
@@ -11,7 +11,7 @@
 	var/hardness = 40 //lower numbers are harder. Used to determine the probability of a hulk smashing through.
 	var/slicing_duration = 100  //default time taken to slice the wall
 	var/sheet_type = /obj/item/stack/sheet/metal
-	var/obj/item/stack/sheet/builtin_sheet = null
+	//var/obj/item/stack/sheet/builtin_sheet = null
```

Comment:
Is there any reason we need to leave this comment in place? If we need it back we can just look into the git history right?